### DISCUSSION TITLE
If I have iceberg tables in S3, can someone walk me through how to access in Fabric

### DISCUSSION THREAD

**Discussion by pawarbi** [apache/incubator-xtable#386] [Q&A]
Hi, I have the Iceberg tables in S3, is there a walkthrough/tutorial on how I can query those in Fabric as delta tables? I would appreciate if you can point me in a direction.

**Comment by sagarlakshmipathy** [comment]
1. What you can do is use XTable to translate the iceberg table metadata to delta table by following the guide [here](https://xtable.apache.org/docs/how-to#steps)
2. Then follow MS Fabric [docs](https://learn.microsoft.com/en-us/fabric/data-factory/connector-amazon-s3) to connect to S3.
3. Then you can query the S3 tables directly using PySpark on Fabric.

Unfortunately there's no guide to address your requirement. If you do manage to get it to work, the community will benefit from it.
